README.md

```diff
@@ -53,78 +53,27 @@ $ docker run --name ecs-agent \
-PS C:\> # Set up configuration
-PS C:\> $ecsExeDir = "${env:ProgramFiles}\Amazon\ECS"
-PS C:\> [Environment]::SetEnvironmentVariable("ECS_CLUSTER", "my-windows-cluster", "Machine")
-PS C:\> [Environment]::SetEnvironmentVariable("ECS_LOGFILE", "${env:ProgramData}\Amazon\ECS\log\ecs-agent.log", "Machine")
-PS C:\> [Environment]::SetEnvironmentVariable("ECS_DATADIR", "${env:ProgramData}\Amazon\ECS\data", "Machine")
-PS C:\> # Download the agent
-PS C:\> $agentVersion = "latest"
-PS C:\> $agentZipUri = "https://s3.amazonaws.com/amazon-ecs-agent/ecs-agent-windows-$agentVersion.zip"
-PS C:\> $zipFile = "${env:TEMP}\ecs-agent.zip"
-PS C:\> Invoke-RestMethod -OutFile $zipFile -Uri $agentZipUri
-PS C:\> # Put the executables in the executable directory.
-PS C:\> Expand-Archive -Path $zipFile -DestinationPath $ecsExeDir -Force
-PS C:\> Set-Location ${ecsExeDir}
-PS C:\> # Set $EnableTaskIAMRoles to $true to enable task IAM roles
-PS C:\> # Note that enabling IAM roles will make port 80 unavailable for tasks.
-PS C:\> [bool]$EnableTaskIAMRoles = $false
-PS C:\> if (${EnableTaskIAMRoles}) {
->> .\hostsetup.ps1
->> }
-PS C:\> # Install the agent service
-PS C:\> New-Service -Name "AmazonECS" `
-        -BinaryPathName "$ecsExeDir\amazon-ecs-agent.exe -windows-service" `
-        -DisplayName "Amazon ECS" `
-        -Description "Amazon ECS service runs the Amazon ECS agent" `
-        -DependsOn Docker `
-        -StartupType Manual
-PS C:\> sc.exe failure AmazonECS reset=300 actions=restart/5000/restart/30000/restart/60000
-PS C:\> sc.exe failureflag AmazonECS 1
+PS C:\> Import-Module ECSTools
+PS C:\> # The -EnableTaskIAMRole option is required to enable IAM roles for tasks.
+PS C:\> Initialize-ECSAgent -Cluster 'windows' -EnableTaskIAMRole
 ```
 
-To run the service, you can do the following:
-```powershell
-Start-Service AmazonECS
-```
+#### Downloading Different Version of ECS Agent
 
-#### As a Process
+In order to download different version of ECS Agent, you can do the following:
 
 ```powershell
-PS C:\> # Set up directories the agent uses
-PS C:\> New-Item -Type directory -Path ${env:ProgramFiles}\Amazon\ECS -Force
-PS C:\> New-Item -Type directory -Path ${env:ProgramData}\Amazon\ECS -Force
-PS C:\> New-Item -Type directory -Path ${env:ProgramData}\Amazon\ECS\data -Force
-PS C:\> # Set up configuration
-PS C:\> $ecsExeDir = "${env:ProgramFiles}\Amazon\ECS"
-PS C:\> [Environment]::SetEnvironmentVariable("ECS_CLUSTER", "my-windows-cluster", "Machine")
-PS C:\> [Environment]::SetEnvironmentVariable("ECS_LOGFILE", "${env:ProgramData}\Amazon\ECS\log\ecs-agent.log", "Machine")
-PS C:\> [Environment]::SetEnvironmentVariable("ECS_DATADIR", "${env:ProgramData}\Amazon\ECS\data", "Machine")
-PS C:\> # Set this environment variable to "true" to enable IAM roles.  Note that enabling IAM roles will make port 80 unavailable for tasks.
-PS C:\> [Environment]::SetEnvironmentVariable("ECS_ENABLE_TASK_IAM_ROLE", "false", "Machine")
 PS C:\> # Download the agent
```

Comment:
The module will do this as well. Maybe add two examples like this?
"
```powershell
Initialize-ECSAgent -Cluster 'windows' -EnableTaskIAMRole -Version 'v1.20.4'
```
*or*
```powershell
Initialize-ECSAgent -Cluster 'windows' -EnableTaskIAMRole -Version 'latest'
```
"